src/main/groovy/wooga/gradle/appcenter/AppCenterPlugin.groovy

```diff
@@ -66,55 +65,34 @@ class AppCenterPlugin implements Plugin<Project> {
+//            {
+//                def result = (List<Map<String, String>> )it
+//                if (result == null){
+//                    result =it.split(',').collect { ["name": it.trim()] }
+//                }
+//                result
+//            }))
+
         extension.defaultDestinations.set(project.provider({
-            String rawValue = (project.properties[AppCenterConsts.DEFAULT_DESTINATIONS_OPTION]
-                    ?: System.getenv()[AppCenterConsts.DEFAULT_DESTINATIONS_ENV_VAR]) as String
+            String rawValue = (project.properties[AppCenterConventions.DEFAULT_DESTINATIONS_OPTION]
+                    ?: System.getenv()[AppCenterConventions.DEFAULT_DESTINATIONS_ENV_VAR]) as String
 
             if (rawValue) {
                 return rawValue.split(',').collect { ["name": it.trim()] }
             }
 
-            AppCenterConsts.defaultDestinations
-        }))
-
-        extension.apiToken.set(project.provider({
-            (project.properties[AppCenterConsts.API_TOKEN_OPTION]
-                    ?: System.getenv()[AppCenterConsts.API_TOKEN_ENV_VAR]) as String
-        }))
-
-        extension.owner.set(project.provider({
-            (project.properties[AppCenterConsts.OWNER_OPTION]
-                    ?: System.getenv()[AppCenterConsts.OWNER_ENV_VAR]) as String
-        }))
-
-        extension.applicationIdentifier.set(project.provider({
-            (project.properties[AppCenterConsts.APPLICATION_IDENTIFIER_OPTION]
-                    ?: System.getenv()[AppCenterConsts.APPLICATION_IDENTIFIER_ENV_VAR]) as String
-        }))
-
-        extension.publishEnabled.set(project.provider({
-            String rawValue = (project.properties[AppCenterConsts.PUBLISH_ENABLED_OPTION]
-                    ?: System.getenv()[AppCenterConsts.PUBLISH_ENABLED_ENV_VAR]) as String
-
-            if (rawValue) {
-                return (rawValue == "1" || rawValue.toLowerCase() == "yes" || rawValue.toLowerCase() == "y" || rawValue.toLowerCase() == "true")
-            }
-            AppCenterConsts.defaultPublishEnabled
-        }))
-
-        extension.retryTimeout.set(project.provider({
-            String rawRetryTimout = (project.properties[AppCenterConsts.RETRY_TIMEOUT_OPTION]
-                    ?: System.getenv()[AppCenterConsts.RETRY_TIMEOUT_ENV_VAR]) as String
-
-            (rawRetryTimout) ? Long.parseLong(rawRetryTimout) : AppCenterConsts.defaultRetryTimeout
+            AppCenterConventions.defaultDestinations
         }))
 
-        extension.retryCount.set(project.provider({
-            String rawRetryCount = (project.properties[AppCenterConsts.RETRY_COUNT_OPTION]
-                    ?: System.getenv()[AppCenterConsts.RETRY_COUNT_ENV_VAR]) as String
-
-            (rawRetryCount) ? Integer.parseInt(rawRetryCount) : AppCenterConsts.defaultRetryCount
-        }))
+        // TODO: Should these still call set? Or we can invoke convention here?
```

Comment:
convention is good here. There was no convention in gradle 4.10 and I kept it backwards compatible.